src/routes/+page.svelte

```diff
@@ -357,16 +349,116 @@
             </div>
         </div>
     </div>
+
+</section>
+
+<section
+    class="container flex flex-col justify-center gap-y-20 px-6 lg:px-20 py-32"
+>
+    <div class="max-w-[320px] max-w-[480px]">
```

Comment:
you have two `max-w` for the same element. Pick one.

also we shouldn't be using arbitrary spacing. Extend the tailwind config to add the values you need